backend/core/test/jurisdictions/jurisdictions.e2e-spec.ts

```diff
@@ -71,4 +71,31 @@ describe("Jurisdictions", () => {
   afterAll(async () => {
     await app.close()
   })
+
+  it(`should create and return a new jurisdiction by name`, async () => {
+    const res = await supertest(app.getHttpServer())
+      .post(`/jurisdictions`)
+      .set(...setAuthorization(adminAccesstoken))
+      .send({ name: "test2" })
+      .expect(201)
+    expect(res.body).toHaveProperty("id")
+    expect(res.body).toHaveProperty("createdAt")
+    expect(res.body).toHaveProperty("updatedAt")
+    expect(res.body).toHaveProperty("name")
+    expect(res.body.name).toBe("test2")
+
+    const getById = await supertest(app.getHttpServer())
+      .get(`/jurisdictions/byName/${res.body.name}`)
+      .set(...setAuthorization(adminAccesstoken))
```

Comment:
We should be able to get this without the token now. Same with the existing gets above.

Reply:
updated 👍